src/helpers/queryByAPI.js

```diff
@@ -82,20 +84,23 @@ export const queryByTestId = (instance: ReactTestInstance) =>
     }
   };
 
-export const queryAllByName = (instance: ReactTestInstance) => (
-  name: string | React.ComponentType<any>
-) => {
-  logDeprecationWarning('queryAllByName', 'getAllByName');
```

Comment:
we've long-deprecated ByName matchers, so I'd like to actually remove this API, not move it to UNSAFE_